kyaml/setters2/settersutil/settercreator.go

```diff
@@ -165,3 +173,36 @@ func validateSchemaTypes(sc *spec.Schema) error {
 	}
 	return nil
 }
+
+func (c SetterCreator) validateSetterInfo() error {
+	openapi.ResetOpenAPI()
```

Comment:
We can't reset the openapi schema here. If the user has read the schema from a file or fetched it from the cluster, this will reset it with the builtin schema

Reply:
I expected this comment. Setter commands in their current form doesn't use k8s schema. They only need setters schema. So had to do this so that only the setter schema for the current package is only loaded for validation.

Reply:
Fixed it. Added a new method to delete schema present in file from global schema so that for every package only the setter schema related to it is loaded and previous package setters schema is cleaned up.